content/applications/productivity/iot/config/troubleshooting.rst

```diff
@@ -99,6 +99,230 @@ Printer`. Follow the steps and select the *make* and *model* corresponding to th
+  `_`, `/`, `#`, or ` ` (space character).
+- `IMC`: This stands for Image Mode Column (the simplified name for `ESC *`).
+- `param_1`: This stands for the specific parameter:
+
+  - `SCALE<X>`: Scale of the picture (with the same aspect ratio). `X` should be an integer
+    describing the scale percentage that should be used.
+
+    .. example::
+       `100` is the original size, `50` is half the size, `200` is twice the size.
+
+  - `LDV`: *Low Density Vertical* (will be set to *High Density Vertical* if not specified).
+  - `LDH`: *Low Density Horizontal* (will be set to *High Density Horizontal* if not specified).
+
+    .. note::
+       *Density* parameters might need to be configured in a particular way depending on the printer
+       model.
+
+.. seealso::
+   Visit `Epson's ESC * documentation
+   <https://reference.epson-biz.com/modules/ref_escpos/index.php?content_id=88>`_ and click on the
+   printer model printer in the table above to see if the printer should set these parameters.
+
+.. example::
+   The following are examples of proper and improper name formatting:
+
+   Proper name formatting:
+
+   - `EPSONTMm30II__IMC__`
+   - `EPSON_TM_U220__IMC_LDV_LDH_SCALE80__`
+
+   Improper name formatting (this will not prevent printing, but the result might not have the
+   expected printed output):
+
+   - `EPSON TMm 30II` -> The name can't have spaces.
+   - `EPSONTMm30II` -> The name itself is correct, but it won't use `ESC *`.
+   - `EPSONTMm30II__IMC` -> This name is missing the end `__`.
+   - `EPSONTMm30II__IMC_XDV__` -> The parameter `XDV` does not match any existing parameters.
+   - `EPSONTMm30II__IMC_SCALE__` -> The parameter `SCALE` is missing the scale value.
+
+After setting the name of the printer with the appropriate naming convention, click
+:guilabel:`Continue`. Next, set the :guilabel:`Make` value to :guilabel:`Raw` and for the
+:guilabel:`Model` value, set this to :guilabel:`Raw Queue (en)`.
+
+After completing these steps, click :guilabel:`Add Printer`. If everything was done correctly, the
+page should redirect to the *Banners* page.
+
+At this point the printer should have been created, now the :abbr:`IoT (Internet of Things)` box
+just needs to detect it and then sync to Odoo's server (this could take a few minutes).
+
+Once the printer is visible on the Odoo database, don't forget to choose it in the :abbr:`PoS (Point
+of Sale)`configuration as the :abbr:`IoT (Internet of Things)` printer. Navigate to
+:menuselection:`Pos App --> Settings --> Connected Devices --> IoT Box --> Receipt Printer -->
+Save`.
+
+.. note::
+   If the printer was set up incorrectly (it is still printing random text or the printed receipt is
+   too big or small) it cannot be modified via the printer name with *CUPS*. Instead the above
+   process can be repeated to set up another printer from scratch to create one with modified
+   parameters.
+
+.. spoiler::
```

Comment:
What do you think of adding an h5 header titled "Example: configure TM-U220B printer"? Also the info you shared in this spoiler block seems too valuable to hide in a hidden block like this. Of the contents inside this block, is there a way to take 1 or 2 screenshots out of here to place in an example admonition block in the earlier sections? (:

Reply:
I added an H7 heading to this section. I will leave it as a spoiler because it is too specific to general troubleshooting and takes up too much real estate. This is a creative way to use the spoiler admonition block and hide unnecessary info. Because this is a practical example, the images are justified.